lib/client/api.go

```diff
@@ -271,9 +275,12 @@ type Config struct {
 	// (not currently implemented), or set to 'none' to suppress browser opening entirely.
 	Browser string
 
-	// UseLocalSSHAgent will write user certificates to the local ssh-agent (or
-	// similar) socket at $SSH_AUTH_SOCK.
-	UseLocalSSHAgent bool
+	// AddKeysToAgent specifies how the client handles keys.
+	//	auto - will attempt to add keys to agent if the agent supports it
+	//	only - attempt to load keys into agent but don't write them to disk
+	//	on - attempt to load keys into agent
+	//	off - do not attempt to load keys into agent
+	AddKeysToAgent string
```

Comment:
This leaves out `AddKeysToAgent` being empty - is zero value also useful or does it have to set (and corr. validated)?

Reply:
The zero value doesn't make sense here since I'd rather have it explicitly be auto as a default. If this flag isn't provided, tsh cli will default this to auto.

Reply:
It also validates the input.